backbone/backbone-global.d.ts

```diff
@@ -147,7 +153,7 @@ declare namespace Backbone {
         /*private*/ set(attributeName: string, value: any, options?: ModelSetOptions): Model;
         set(obj: any, options?: ModelSetOptions): Model;
 
-        changedAttributes(attributes?: any): any[];
+        changedAttributes(attributes?: any): any | boolean;
         clear(options?: Silenceable): any;
```

Comment:
```any | boolean``` is the same as ```any``` - I suggest just using ```any```. This gives the illusion of stricter type checking without actually providing it. A JSDoc comment describing the return value may be more helpful in practice.